xml/ns-System.Net.Mail.xml

```diff
@@ -1,12 +1,20 @@
 <Namespace Name="System.Net.Mail">
+  <Attributes>
```

Comment:
This is a reflection-only portion of the XML. Please revert this change for the new lines 2 through 6.